grafanalib/core.py

```diff
@@ -427,6 +427,27 @@ def to_json_data(self):
         }
 
 
+@attr.s
+class SqlTarget(Target):
+    """
+    Metric target to support SQL queries
+    """
+
+    rawSql = attr.ib(default="")
+    rawQuery = attr.ib(default=True)
```

Comment:
@JamesGibo I added the rawQuery, but in the docstring I highlighted that it should be True , now the by default is True.